src/OpenLoco/src/Localisation/Formatting.h

```diff
@@ -0,0 +1,148 @@
+#pragma once
+
+#include "StringManager.h"
```

Comment:
does this need to be included?

Reply:
I don't know how I feel about forward declaring a type so often, more cleanups will be done but I haven't quite decided where string_id would be best placed.